src/nvim/fileio.c

```diff
@@ -1423,29 +1423,36 @@ readfile (
               }
             }
           }
-          if (enc_utf8) {               /* produce UTF-8 */
+          if (enc_utf8) {               // produce UTF-8
```

Comment:
`enc_utf8` is always true in nvim, so the else branch ("produce latin1") can be deleted.

Reply:
done